rad/src/lib.rs

```diff
@@ -42,6 +43,20 @@ pub struct RADRequestExecutionReport {
     pub tally: RadonReport<RadonTypes>,
 }
 
+/// Auxiliar function that returns the current active wips
+pub fn current_active_wips() -> ActiveWips {
+    let mut active_wips = HashMap::<String, Epoch>::default();
+    active_wips.insert("WIP0008".to_string(), 0);
+    active_wips.insert("WIP0009-0011-0012".to_string(), 0);
+    active_wips.insert("THIRD_HARD_FORK".to_string(), 0);
+    active_wips.insert("WIP0014-0016".to_string(), 0);
+
+    ActiveWips {
+        active_wips,
+        block_epoch: 500000,
+    }
+}
+
```

Comment:
Let me suggest the addition of a comment mentioning that the activation epoch for all of these is 0 just because they did not introduce any changes that affected RADON or the RAD engine.

Reply:
An alternative would be to store a "radon version" number instead of storing the individual WIPs, and then use the WIPs to set the radon version where needed.